yb-voyager/test/containers/testcontainers.go

```diff
@@ -48,15 +49,20 @@ func NewTestContainer(dbType string, containerConfig *ContainerConfig) TestConta
 	registryMutex.Lock()
 	defer registryMutex.Unlock()
 
+	return setUpContainer(dbType, containerConfig, false)
+}
+
+func setUpContainer(dbType string, containerConfig *ContainerConfig, forLive bool) TestContainer {
 	// initialise containerConfig struct if nothing is provided
 	if containerConfig == nil {
 		containerConfig = &ContainerConfig{}
 	}
 	setContainerConfigDefaultsIfNotProvided(dbType, containerConfig)
 
+	registryMap := lo.Ternary(forLive, containerRegistryForLive, containerRegistry)
 	// check if container is already created after fetching default configs
 	containerName := fmt.Sprintf("%s-%s", dbType, containerConfig.DBVersion)
```

Comment:
New logic for container name - include Live aspect if present.
Then we can just reuse the same registry map, will be simpler.